crates/hyperswitch_interfaces/src/consts.rs

```diff
@@ -5,3 +5,9 @@ pub const NO_ERROR_MESSAGE: &str = "No error message";
 
 /// No error code string const
 pub const NO_ERROR_CODE: &str = "No error code";
+
+/// Accepted format for request
+pub const ACCEPT_HEADER: &str = "text/html,application/json";
```

Comment:
This value is specific to the Itau Bank connector only though, may not be applicable to more connectors. I'd _prefer_ this being in the Itau Bank module.